apps/docs/.vitepress/components/PageContributors.vue

```
@@ -0,0 +1,59 @@
+  return contributors;
+});
+</script>
+
+<template>
+  <div v-if="isComponentPage && contributors.length">
+    <div class="page-contributors-title">
+      {{ contributorTitle }}
+    </div>
+    <div class="page-contributors-list">
+      <a v-for="contributor in contributors" :key="contributor.login" class="page-contributors-item"
+        :href="contributor.homeUrl" target="_blank">
```

ESLint

[error] Expected indentation of 9 spaces but found 8 spaces.

(vue/html-indent)